src/js/plugins/plugin.mobile_nav.js

```diff
@@ -148,10 +178,20 @@ BookReader.prototype.buildToolbarElement = (function (super_) {
  */
 BookReader.prototype.buildMobileDrawerElement = function() {
   let experimentalHtml = '';
+  //builds filters checkbox html
   if (this.enableExperimentalControls) {
     experimentalHtml = `
         <p class="DrawerSettingsTitle">Experimental (may not work)</p>
-        <button class="BRaction default high-contrast-button">Toggle high contrast</button>
+        <div class="BRcheckbox-group-filters">
+          <input type="checkbox" class="BRcheckbox-filters" id="filter1">
+          <label for="filter1" class="BRcheckbox-label-filters">Grayscale</label><br>
+          <input type="checkbox" class="BRcheckbox-filters" id="filter2">
+          <label for="filter2" class="BRcheckbox-label-filters">High brightness</label><br>
+          <input type="checkbox" class="BRcheckbox-filters" id="filter3">
+          <label for="filter3" class="BRcheckbox-label-filters">Inverted</label><br>
```

Comment:
Might want to change this to "Inverted (Night mode)" for folks who might not understand what inverted means.